src/index.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+
+<html lang="en">
+	<head>
+	 	<meta charset="utf-8">
+ 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" type="text/css" href="styles.css"/>
+        <link rel="shortcut icon" href="favicon.ico" type="image/x-icon"/>
+		<script src="https://unpkg.com/vue"></script>
```

Comment:
Let's use the minified version of Vue https://unpkg.com/vue@2.4.2/dist/vue.min.js